frontend/app/templates/components/account-form.hbs

```diff
@@ -43,23 +40,23 @@
 
                   <div class="form-group col-sm-6">
                     <div class="row">
-                      <Input class="form-control " @value={{this.changeset.cleartextPassword}} @name="cleartextPassword" @type="password"/>
+                      <Input class="form-control mb-10" @value={{mut this.changeset.cleartextPassword}} @name="cleartextPassword" @type="password" @key-up={{action "updatePasswordScore"}}/>
+                      {{ember-progress-bar progress=this.passwordScore options=(hash strokeWidth=3 duration=1)}}
                     </div>
                   </div>
 
                   <div class="form-group col-xs-12 col-sm-5 pull-right">
                     <div class="row">
                       <button class="btn btn-secondary btn-block" {{on "click" this.setRandomPassword}}>{{t "randomPassword"}}</button>
+                      <label>{{this.passwordLabel}}</label>
```

Comment:
I would place the Label on one line right under the progress bar, instead under the Random Button.

Reply:
Sure.